pages/1.11/security/ent/secrets/create-secrets/index.md

```diff
@@ -90,7 +110,11 @@ The contents of the file (referred to below as `my-secret.txt`) can be any text
 1. Use the following command to create the new secret.
 
   ```bash
-  dcos security secrets create -f my-secret.txt developer/my-secret
+  dcos security secrets create --text-file my-secret.txt developer/my-secret
   ```
 
+  The value of the `my-secret.txt` should be a text encoded in UTF-8.
```

Comment:
"The value of" do you mean "The content of"?

> the `my-secret.txt

Either remove "the" or use "file" afterwards

> "should be a text"

Remove "a"